tests/Client.test.js

```
@@ -19,26 +19,50 @@ describe("Client", () => {
   test("no transports specified", async () => {
     expect.assertions(1);
 
+    let error;
     try {
       await API_CLIENT.organizations.list({
         transportMode: []
       });
-    } catch (error) {
-      expect(error).toBeInstanceOf(EndpointUndefinedError);
+    } catch (e) {
+      error = e;
     }
+    expect(error).toBeInstanceOf(EndpointUndefinedError);
```

🍖  The problem with the error handling above was not caught in tests because our expectations were only checked if an error is thrown. If no error at all is thrown the test would silently pass.

I've updated all of these tests to use one of the patterns to achieve this for async methods – move `expect` out of the catch block.